commit f31324a23dc2301cf2e664118c27b5c6be6bcda0
Author: Robert Mustacchi <rm@joyent.com>
Date:   2019-05-08T20:43:25+00:00 (5 months ago)
    
    OS-7787 make clean fails if ./tools/builder is already clean

diff --git a/tools/builder/Makefile b/tools/builder/Makefile
index 292d62eb..4b9bce31 100644
--- a/tools/builder/Makefile
+++ b/tools/builder/Makefile
@@ -1,5 +1,5 @@
 #
-# Copyright (c) 2010 Joyent Inc., All rights reserved.
+# Copyright 2019 Joyent, Inc.
 #
 
 CC=gcc
@@ -21,4 +21,4 @@ users.c:
 	./build_users_c.sh $(SRC_ROOT)/overlay/generic > users.c.tmp && mv users.c.tmp users.c
 
 clean:
-	rm $(TARGETS) $(OBJS)
+	rm -f $(TARGETS) $(OBJS)
